app/views/unattended/report_templates/jobs_invocations.erb

```diff
@@ -0,0 +1,42 @@
+<%#
+name: Jobs Invocations report template
+snippet: false
+template_inputs:
+- name: job_id
+  required: true
+  input_type: user
+  description: ID of job to report
+  advanced: false
+  value_type: plain
+  resource_type: JobInvocation
+  hidden_value: false
+- name: hosts
+  required: false
+  input_type: user
+  description: Field for filter hosts of job invocation. Leave blank for all hosts.
+  advanced: false
+  value_type: search
+  resource_type: Host
+  hidden_value: false
+model: ReportTemplate
+require:
+- plugin: foreman_remote_execution
+￼ version: 4.2.2
+-%>
```

Comment:
There is some weird character, could you check what is it?

Reply:
I've check that and I didn't find anything unusual character.